pkg/gce-pd-csi-driver/controller.go

```diff
@@ -921,6 +961,14 @@ func (gceCS *GCEControllerServer) createPDSnapshot(ctx context.Context, project
 			if gce.IsGCEError(err, "notFound") {
 				return nil, status.Errorf(codes.NotFound, "Could not find volume with ID %v: %v", volKey.String(), err.Error())
 			}
+			// Emit metric for error
+			if strings.Contains(sourceDisk.GetPDType(), "hyperdisk") {
+				gceCS.Metrics.RecordOperationErrorMetrics(hyperdiskDriverName, "/csi.v1.Controller/CreateSnapshot", err, sourceDisk.GetPDType())
```

Comment:
Nit: the "/csi.v1.Controller/" is a constant repeat, we can also construct it in the RecordOperationErrorMetrics func.

const prefixForMethodName = "/csi.v1.Controller/"

...
func (mm *MetricsManager) RecordOperationErrorMetrics(
	operationName string,
	operationErr error,
	diskType string) {
        // Add code to decide driverName based on diskType
	pdcsiOperationErrorsMetric.WithLabelValues(driverName, prefixForMethodName + operationName, CodeForErrors(operationErr), diskType).Set(1.0)
}